drivers/HwWatchdog.h

```diff
@@ -0,0 +1,94 @@
+/* mbed Microcontroller Library
```

Comment:
These files would probably make more sense in the platform directory with a name like `mbed_watchdog_mgmt.h`. Camel case is used for classes and the file that contains them while for non-class code the convention is to use underscores.